account_financial_report_qweb/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# © 2015 Yannick Vaucher
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).-
+from . import models
```

Comment:
Damn, I have always set the same order as the order of use. :-/

models, fields, api